test/cli/bitswap.js

```diff
@@ -108,8 +66,5 @@ describe('bitswap', () => runOn((thing) => {
     expect(out).to.include(new CID(key1).toBaseEncodedString('base64'))
   })
 
-  it('unwant', async function () {
-    const out = await ipfs('bitswap unwant ' + key0)
-    expect(out).to.eql(`Key ${key0} removed from wantlist\n`)
-  })
+  // TODO: missing unwant with cid-base arg
```

Comment:
There are quite a lot of TODO being added here - how will they be resolved?  By adding tests to these suites or the interface tests?